inventory/group_vars/fr.yml

```diff
@@ -24,3 +24,4 @@ users_sysadmin:
   - pau
   - enricostn
   - luis
+  - kristina
```

Comment:
👍 Welcome @kristinalim

Reply:
Thank you, @pacodelaluna! :slightly_smiling_face: